### PR TITLE
Fixed Redirection Bug

### PR DESCRIPTION
**Pull request by siddhant1**

### CODE REVIEW

**Comment by siddhant1** [comment]
This is also leading to popups.

File: public/index.html [hunk 1]
```diff
@@ -28,6 +28,7 @@
     You need to enable JavaScript to run this app.
   </noscript>
   <div id="root" style="max-width: 100vw; min-height: 100vh;"> </div>
+  <div id="links" style="display:none"> </div>
```
Comment (alexdor) [inline public/index.html]:
```suggestion
```

File: src/helpers/helpers.ts [hunk 2]
```diff
@@ -20,9 +20,21 @@ export const customPageView = (url: string) => {
...
+    return;
+  } else {
+    outboundLink({ label: url }, () => {
+      const link = document.createElement("a");
+      link.setAttribute("href", url);
+      link.target = "_blank";
+      link.style.display = "none";
+      link.setAttribute('rel',"rel: 'noopener noreferrer'")
+
+      document.getElementById("links")!.appendChild(link);
```
Comment (alexdor) [inline src/helpers/helpers.ts]:
```suggestion
      document.body.appendChild(link);
```

File: src/helpers/helpers.ts [hunk 3]
```diff
@@ -20,9 +20,21 @@ export const customPageView = (url: string) => {
...
 };
 
-export const customOutboundLink = (url: string) =>
-  develop
-    ? window.open(url, '_blank')
-    : outboundLink({ label: url }, () => {
-      window.open(url, '_blank')
+export const customOutboundLink = (url: string) => {
+  if (develop) {
+    const newWindow = window.open(url, "_blank");
```
Comment (alexdor) [inline src/helpers/helpers.ts]: could you change this part as well?

**Comment by siddhant1** [comment]
Sure ! I will look into all of them.

**Comment by alexdor** [comment]
@siddhant1 are you still facing popups?

**Comment by siddhant1** [comment]
Yes, in firefox

**Comment by siddhant1** [comment]
Fixes #99

**Pull request merged**